shared/chat/conversation/messages/exploding-explainer-dialog/index.js

```diff
@@ -30,7 +30,7 @@ export const ExplodingExplainerElement = (props: Props) => (
       </Kb.Text>
       <Kb.Box style={styles.buttonBox}>
         <Kb.Button
-          type="Primary"
+          
```

Comment:
Hm, are these lines adding a new empty whitespace line? Diff's a bit confusing.

Reply:
urgh, I did find+replace and I guess VS code does this. I can try to fix it before merge, does it make this too hard to review?

Reply:
Not a big deal for the review, fixing before merge is fine!